tests/xharness/Program.cs

```diff
@@ -1,13 +1,34 @@
 ﻿using System;
-using System.Collections.Generic;
 using Mono.Options;
-using Xharness.Execution;
-using Xharness.Utilities;
+using Microsoft.DotNet.XHarness.iOS.Shared.Utilities;
+using Microsoft.DotNet.XHarness.iOS.Shared;
 
 namespace Xharness {
 	class MainClass {
 		public static int Main (string [] args)
 		{
+			Microsoft.DotNet.XHarness.iOS.Shared.SdkVersions.OverrideVersions (
```

Comment:
@mandel-macaque is this the only entrypoint for the Xharness code or do we need to call it from Jenkins somehow as well?

Reply:
AFAIK this is the only entry point for xharness. Jenkins will call the main from xharness.exe. @rolfbjarne should be able to confirm, but I believe is the case.

Reply:
Main is the only entry point, xharness is only used as an executable, not a library.